### PR TITLE
Fix Uppercase issue

### PR DESCRIPTION
**Pull request by alexvandesande**
* Converts all searched names to lowercase
* Fixes a gas explainer issue

### CODE REVIEW

File: app/imports/templates/layout/header.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ Template['layout_header'].onRendered(function() {
 
 Template['layout_header'].events({
   'input #search-input': function(event, template) {
-    event.target.value = event.target.value.replace(/[\.\s\_\/\\\@]*/g,'');
+    event.target.value = event.target.value.replace(/[\.\s\_\/\\\@]*/g,'').toLowerCase();
```
Comment (Arachnid) [inline app/imports/templates/layout/header.js:31]: Any plans to start using the normalize func instead?

**Review by Arachnid — APPROVED** [review]

File: app/imports/templates/components/nameStatus.html [hunk 2]
```diff
@@ -168,6 +168,9 @@
 <template name="status-reveal">
     <div>
         <h1> <strong>{{name}}</strong> waiting for bids to be revealed </h1>
+        <p>
+          After an auction ends it's 72h period of bidding, all bidders have 48h to reveal their bids or forfeit their whole deposit. During this period, no new bids can be made. If no bids are revealed, the name can be requested again after this period ends.
```
Comment (Arachnid) [inline app/imports/templates/components/nameStatus.html]: s/it's/its/

**Review by Arachnid — APPROVED** [review]

File: app/imports/templates/layout/header.js (hunk 1 above)
Comment (alexvandesande) [reply app/imports/templates/layout/header.js:31]: the eth-ens-js library passes the name through it. Or at least it should. This is just a basic function to clean up a bit.

Comment (Arachnid) [reply app/imports/templates/layout/header.js:31]: There must be somewhere it's not doing that, though, or this bug couldn't exist.

**Pull request merged**